src/containers/MovieContainer.test.js

```diff
@@ -0,0 +1,14 @@
+// import React from 'react';
+// import { shallow } from 'enzyme';
+// import MovieContainer from './MovieContainer';
+//
+// // describe('MovieContainer', () => {
+// //   it('should match the snapshot with all data successfully passed', () => {
+// //     const wrapper = shallow(
+// //       <MovieContainer
+// //       allMovies={[]}
+// //     />);
+// //
+// //     expect(wrapper).toMatchSnapshot();
+// //   });
+// // });
```

Comment:
Instead of commenting out the test, I think it might be better to use `it.skip` for right now.

Reply:
Good idea @peeratmac.  I wouldn't have thought of that.

Reply:
Will do! Thanks, Peerat